x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/async/AsyncTaskIndexService.java

```diff
@@ -327,18 +301,29 @@ private void updateResponse(
         }
     }
 
-    /**
-     * Update the initial stored response with a failure
-     */
-    private void updateStoredResponseWithFailure(
```

Comment:
Inlined as this is a somewhat redundant method and now was only used in a single spot.